src/Tribe/RSVP/Attendance_Totals.php

```diff
@@ -42,13 +42,23 @@ public function print_totals() {
 		$going = $this->get_total_going();
 		$not_going = $this->get_total_not_going();
 
-		echo "
+		$html = "
 			<ul>
 				<li> <strong>$total_rsvps_label</strong> $total_rsvps </li>
 				<li> <strong>$going_label</strong> $going </li>
 				<li> <strong>$not_going_label</strong> $not_going </li>
+				$attendance_lines_after
```

Comment:
Nope, ignore my previous comment: the filter is in place (the `tribe_tickets_rsvp_print_totals_html` one) and that's a left-over that var is a leftover that should be removed.